assets/home/components/search-bar.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import {gettext} from 'utils';
+
+export class SearchBar extends React.PureComponent {
+    render() {
+        return (
+            <nav className="content-bar navbar justify-content-start flex-nowrap flex-sm-wrap">
```

Comment:
Please put nav inside the block: 
`<section class="content-header"><nav ...>...</nav></section> `
so we have proper search bar height (56px).

Reply:
That's it thanks. If it is not too much of work can you also add 'x' for clearing the search input, same as in wire view?